spyderlib/plugins/shortcuts.py

```diff
@@ -415,14 +417,17 @@ def save(self):
         set_shortcut(self.context, self.name, self.key)
 
 
-CONTEXT, NAME, SEQUENCE = [0, 1, 2]
+CONTEXT, NAME, SEQUENCE, SEARCH_SCORE = [0, 1, 2, 3]

```

Comment:
I guess you are using Qt5 ?, and the order of the sort is a mandatory argument there, simple fix,